sparkctl/cmd/create.go

```diff
@@ -177,6 +181,29 @@ func createSparkApplication(app *v1beta2.SparkApplication, kubeClient clientset.
 
 	fmt.Printf("SparkApplication \"%s\" created\n", app.Name)
 
+	if WaitTillCompletion {
+		for {
+			appStatus, err := getSparkApplication(app.Name, crdClient)
+			if err != nil {
+				return fmt.Errorf("failed to get SparkApplication %s: %v", app.Name, err)
+			}
+
+			fmt.Printf("SparkApplication \"%s\" status: \"%s\"\n", app.Name,appStatus.Status.AppState.State)
```

Comment:
You can use %q instead of %s, which adds double quotes for you I believe.

Reply:
No problem, I was only following the code-style that I already found in the file (ex: line 182)